doc/swagger.yml

```diff
@@ -409,6 +409,29 @@ paths:
           description: OK
 
   '/{idBranch}/vector':
+    get:
+      tags:
+        - vector
+      summary: "Récuperation d'un vecteur annexe à partir de son nom"
+      description: ""
+      parameters:
+        - in: path
+          name: idBranch
+          description: l'identifiant de la branche
+          required: true
+          schema:
+            type: integer
+        - in: query
+          name: name
+          description: le nom de la couche vecteur
+          required: true
+          schema:
+            type: string
+          example: Remarques
+      responses:
+        '200':
+          description: OK
+  
```

Comment:
Le vecteur  Remarques exporté (en geojson) doit pouvoir être réimporté comme vecteur d'alertes dans une vue itowns, mais ce geojson est bien lisible sur QGIS, mais il n'est pas accepté par itowns - avec ce message d'erreur : `Uncaught [Error: Type of file not supported, please add it using DragNDrop.register at Object.addDnDFiles…]`

Reply:
Corrigé